framework/src/main/java/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -307,7 +307,21 @@ public Receiver visit(NameExpr expr, FlowExpressionContext context) {
             }
 
             if (fieldElem != null && fieldElem.getKind() == ElementKind.FIELD) {
-                return getReceiverField(s, context, originalReceiver, fieldElem);
+                FieldAccess fieldAccess =
+                        (FieldAccess) getReceiverField(s, context, originalReceiver, fieldElem);
+                String className = fieldAccess.getReceiver().getType().toString();
+                Element scopeClassElement =
```

Comment:
Use `org.checkerframework.javacutil.TypesUtils#getTypeElement` to go from a `TypeMirror` to an `TypeElement`.